_inc/client/plans/single-product-backup.jsx

```diff
@@ -238,19 +278,21 @@ class SingleProductBackupBody extends React.Component {
 						/>
 					) ) }
 				</div>
-				{ upgradeLinks &&
-					upgradeLinks[ selectedBackupType ] &&
-					upgradeTitles[ selectedBackupType ] && (
-						<div className="single-product-backup__upgrade-button-container">
-							<Button
-								href={ upgradeLinks[ selectedBackupType ] }
-								onClick={ this.handleUpgradeButtonClick( selectedBackupType ) }
-								primary
-							>
-								{ upgradeTitles[ selectedBackupType ] }
-							</Button>
-						</div>
-					) }
+				<p>
+					<em>
+						<ProductSavings
+							selectedBackup={ selectedBackup }
+							billingTimeFrame={ billingTimeFrame }
+							currencyCode={ currencyCode }
+						/>
+					</em>
+				</p>
+				<UpgradeButton
```

Comment:
Something seems off since we started using this component for the upgrade button:

![](https://cldup.com/rYNgAVSj_2.png)

Reply:
I guess this is because #14231 is not yet merged and so the `PlanPrice` disregards `inline` prop.

Reply:
Yes, this should be resolved now that I've rebased!